OpenOversight/app/models.py

```diff
@@ -33,9 +33,10 @@ class Department(db.Model):
     name = db.Column(db.String(255), index=True, unique=True, nullable=False)
     short_name = db.Column(db.String(100), unique=False, nullable=False)
     unique_internal_identifier_label = db.Column(db.String(100), unique=False, nullable=True)
+    is_active = db.Column(db.Boolean, default=True)
```

Comment:
I think we don't want this to be nullable, since a department is either active or disabled